pkg/service/oidc4ci/oidc4ci_service.go

```diff
@@ -289,6 +292,10 @@ func (s *Service) ValidatePreAuthorizedCodeRequest(
 	}
 	tx.State = newState
 
+	if tx.PreAuthCodeExpiresAt.UTC().Before(time.Now().UTC()) {
+		return nil, resterr.NewCustomError(resterr.OIDCTxNotFound, fmt.Errorf("expired pre-authorize code"))
```

Comment:
I think - for security reasons which shouldn't tell if code is expired.
Section 5.2 RFC6749 https://www.rfc-editor.org/rfc/inline-errata/rfc6749.html

Reply:
Fixed